### PR TITLE
IGAPP-506: Malte translations

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by maxammann** [comment]
If I did not fuck up this should add the new translations. This was quite challenging because we changed fa to pes, and also removed a lot of translation keys. Merging took over 1 hour :D

File: translations/translations.json [hunk 1]
```diff
@@ -1267,26 +2067,8 @@
       "thanksMessage": "Muchas gracias por sus comentarios. Con ellos nos ayuda a mejorar nuestros contenidos y la app {{appName}}. Le invitamos a enviarnos más comentarios.",
       "close": "Cerrar",
       "negativeRating": "Esta página no sirve de ayuda",
-      "positiveRating": "Esta página sirve de ayuda"
```
Comment (klinzo) [inline translations/translations.json:1270]: Should probably not be removed

**Review by klinzo — APPROVED** [review]
No changes of the aschaffenburg overrides is probably correct I guess. Looks quite good, did not se any other problems. Was not that thorough though.

**Review by SteffiStuffel — APPROVED** [review]
Looks good, tested a bit, for missing translations, but couldn't find any. :+1:

**Pull request merged**